Prelude-UIKit/UIButton.swift

```diff
@@ -10,4 +10,16 @@ extension UIButton {
   public func setBackgroundColor(_ backgroundColor: UIColor, for state: UIControlState) {
     self.setBackgroundImage(.pixel(ofColor: backgroundColor), for: state)
   }
+
+  @available(iOSApplicationExtension 10.0, *)
+  public func generateSelectionFeedback() {
+    let notification = UISelectionFeedbackGenerator()
```

Comment:
Two questions:

1. Should we name this `selection` instead of `notification` for less ambiguity?
2. Should we even declare local var or just simply do one liner - `UISelectionFeedbackGenerator().selectionChanged()`